### PR TITLE
Update metadata style check scripts

### PR DESCRIPTION
**Pull request by Luiy0**
### Description

- Update the scripts to include the new `real_time` category for metadata style checks
- Update the `metadata_style_checker` script to exclude the `module-info.java` file for checks
- Tested running the [metadata_style_checker](https://devtopia.esri.com/runtime/common-samples/wiki/Java-Samples-Repo%3A-A-Contribution-Guide#running-locally) locally

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by Luiy0** [comment]
Ready for a first look @jenmerritt, thanks.

**Review by jenmerritt — APPROVED** [review]

**Comment by Luiy0** [comment]
Added script line to exclude the `module-info` file @jenmerritt.

**Comment by Luiy0** [comment]
Hi @Rachael-E, can I get a second review please?

**Review by Rachael-E — APPROVED** [review]
looks ok to me, thanks @Luiy0 !

**Pull request merged**
